src/Pact/Eval.hs

```diff
@@ -291,12 +432,16 @@ reduce (TObject ps t i) =
 reduce (TBinding ps bod c i) = case c of
   BindLet -> reduceLet ps bod i
   BindSchema _ -> evalError i "Unexpected schema binding"
-reduce t@TModule {} = evalError (_tInfo t) "Module only allowed at top level"
+reduce TModule{..} =
+  case _tModuleDef of
```

Comment:
Casing unnecessary, just say "module/interface not allowed ..."

Reply:
Fixed